server/libs/database/src/config/database.config.ts

```diff
@@ -1,17 +1,36 @@
 import { PostgresConnectionOptions } from 'typeorm/driver/postgres/PostgresConnectionOptions';
 import { DataSource } from 'typeorm';
 
-export const databaseConfig: PostgresConnectionOptions = {
+let additionalSSLDatabaseConfig;
+let baseDatabaseConfig: PostgresConnectionOptions = {
+  type: 'postgres',
+  entities: [__dirname + '/../**/*.entity.{js,ts}'],
+  synchronize: false,
+  migrations: [__dirname + '/../migrations/*.{js,ts}'],
+  migrationsRun: true,
+};
+
+let urlBasedDatabaseConfig: PostgresConnectionOptions = {
+  type: 'postgres',
+  url: process.env.DB_URL || 'postgres://postgres:postgres@immich_postgres:5432/immich?sslmode=disable',
```

Comment:
I don't think we need this default value, since we only use these options if DB_URL exists.

Reply:
I have solved this using feedback from @jrasm91 but haven't pushed it yet